RHDEVDOCS-3260 Changed triggers API version to v1beta1 (#37744)
openshift/openshift-docs: modules/op-about-pipelinerun.adoc

// *openshift_pipelines/op-creating-applications-with-cicd-pipelines.adoc

[id="about-pipelinerun_{context}"]
= Pipeline run

A _pipeline run_ is the running instance of a pipeline. It instantiates a pipeline for execution with specific inputs, outputs, and execution parameters on a cluster. A corresponding task run is created for each task automatically in the pipeline run.

All the tasks in the pipeline are executed in the defined sequence until all tasks are successful or a task fails. The `status` field tracks and stores the progress of each task run in the pipeline run for monitoring and auditing purpose.

> The following example shows a pipeline run to execute the `build-and-deploy` pipeline  with relevant resources and parameters:

Reviewer: ```suggestion
The following example runs the `build-and-deploy` pipeline with relevant resources and parameters:
```
- Per the IBM Style Guide, we should avoid using `execute`. 'Use only if a simpler term, such as "run", "issue", "start", or "enter" is inappropriate.' My suggested edit above shortens the sentence by removing implied meaning and avoids using `execute`. 
- Also, extra space deleted
